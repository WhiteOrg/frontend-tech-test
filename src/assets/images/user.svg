<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="none" viewBox="0 0 18 18">
    <path fill="#fff" fill-rule="evenodd" d="M13.5 4.5C13.5 6.986 11.486 9 9 9S4.5 6.986 4.5 4.5 6.514 0 9 0s4.5 2.014 4.5 4.5zM0 15.75c0-2.992 5.996-4.5 9-4.5 3.004 0 9 1.508 9 4.5V18H0v-2.25z" clip-rule="evenodd"/>
</svg>
