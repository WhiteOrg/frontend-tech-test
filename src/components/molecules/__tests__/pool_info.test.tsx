import React from "react";
import renderer from "react-test-renderer";
import PoolInfo from "../pool_info";
import user from "../../../images/user.svg";

describe("MOLECULES: Render testing", () => {
  test("Should render PoolInfo", () => {
    const component = renderer.create(
      <div>
        <PoolInfo title="prize pool" info="Â£25,000.00" />
        <br />
        <PoolInfo title="player" info="359" icon={user} />
      </div>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
