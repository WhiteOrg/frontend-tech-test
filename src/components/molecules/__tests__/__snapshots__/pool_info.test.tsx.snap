// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MOLECULES: Render testing Should render PoolInfo 1`] = `
<div>
  <div
    className="sc-AxirZ svjdj"
  >
    <p
      className="sc-AxjAm hzarxW"
      color="#17dfe7"
      size="small"
    >
      prize pool
    </p>
    <p
      className="sc-AxjAm jPqnYf"
      color="white"
      size="large"
    >
      Â£25,000.00
    </p>
  </div>
  <br />
  <div
    className="sc-AxirZ svjdj"
  >
    <p
      className="sc-AxjAm hzarxW"
      color="#17dfe7"
      size="small"
    >
      player
    </p>
    <div
      className="sc-AxiKw iRXjVE"
    >
      <img
        className="sc-AxhCb ccMvAG"
        src="user.svg"
      />
      <p
        className="sc-AxjAm jPqnYf"
        color="white"
        size="large"
      >
        359
      </p>
    </div>
  </div>
</div>
`;
